eos/form-factors/parametric-kkvdz2022.hh

```diff
@@ -0,0 +1,94 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2022 Danny van Dyk
+ * Copyright (c) 2022 Stephan Kuerten
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#ifndef EOS_GUARD_SRC_FORM_FACTORS_PARAMETRIC_KKVDZ2022_HH
+#define EOS_GUARD_SRC_FORM_FACTORS_PARAMETRIC_KKVDZ2022_HH 1
+
+#include <eos/form-factors/mesonic.hh>
+#include <eos/form-factors/mesonic-processes.hh>
+#include <eos/maths/complex.hh>
+#include <eos/maths/power-of.hh>
+#include <eos/utils/options.hh>
+#include <eos/utils/parameters.hh>
+#include <eos/utils/qualified-name.hh>
+#include <eos/utils/reference-name.hh>
+
+#include <memory>
+
+namespace eos
+{
+    class KKvDZ2022FormFactors :
+        public FormFactors<PToGammaOffShell>
+    {
+        private:
+            double _widthOmega(const double & q2) const;
```

Comment:
if the width and z functions are simple, can we make them inline and move into the header?